documentation/sphinx/source/release-notes.rst

```diff
@@ -60,6 +60,10 @@ Fixes
 * The cluster controller did not do locality aware reads when measuring status latencies. [6.0.12] `(PR #801) <https://github.com/apple/foundationdb/pull/801>`_
 * Storage recruitment would spin too quickly when the storage server responded with an error. [6.0.12] `(PR #801) <https://github.com/apple/foundationdb/pull/801>`_
 * The transaction logs would leak memory when serving peek requests to log routers. [6.0.12] `(PR #801) <https://github.com/apple/foundationdb/pull/801>`_
+* Restoring to the exact version a snapshot ends did not apply mutations done at the final version. [6.0.12] `(PR #787) <https://github.com/apple/foundationdb/pull/787>`_
+* The transaction logs were spilling data unnecessarily. [6.0.12] `(PR #784) <https://github.com/apple/foundationdb/pull/784>`_
```

Comment:
I'm not sure this line will be well understood by people reading the release notes. Maybe we could add that the observable consequence was extra needless disk writes.